### DISCUSSION TITLE
LXQT Reducing RAM Usage

### DISCUSSION THREAD

**Discussion by lxandy578** [lxqt/lxqt#2641] [Q&A]
How can I further reduce RAM usage in LXQT?”

**Comment by tsujan** [comment]
LXQt has no known memory leak and its RAM usage is low. If you see a high memory usage, you might want to check other parts of your system.

**Comment by Frtzqe** [comment]
I guess your question is relevant because your particular system is very limited with RAM: 2 GB or even only 1 GB. 

In case of 2 GB RAM, in general you should not care about reducing the footprint of your system (including LXQt) at all. It really is not needed, or worth it. Basic browsing, opening a spreadsheet, it should all work normally. You could gain a few hundreds MB headroom by replacing Firefox by Brave. 

In case of 1 GB (or less) RAM there are a few things you could do to avoid undeterministic stalling (swapping to disk) or even random crashes:
* don't use any desktop like LXQt or Xfce at all. Just replace it by using plain _openbox_. Generate a correct menu with menumaker. It is bare, but very functional (with a regenerated menu);
* stop using a greeter program (e.g. "systemctl disable --now sddm") and start LXQt manually ("startlxqt"). Note: not all distros support this and you may need to change a script;
* stop the LXQt desktop;
* you could even stop the LXQt panel;
* stop all other modules.

LXQt being modular is a real nice feature, so make use of it! (Session Settings, under Preferences).

You will not gain much by this, but it might just avoid swapping/crashing, if you're lucky.

**Comment by Unkown000X** [comment]
> How can I further reduce RAM usage in LXQT?”

What is your real ram consumption? I recommend you to look at this page 

https://www.linuxatemyram.com/

**Comment by sonichy** [comment]
How big is your LXQT RAM Usage ?

**Reply by isf63** [reply]
On my installation, about 450M. 180M of that is `pcmanfm-qt --desktop` and `lxqt-panel`.

If I had to guess (more than?) half of the RAM usage comes for Qt overhead - `lxqt-panel` uses 70M yet `lxqt-runner` (despite being orders-of-magnitude simpler) uses 45M.

**Reply by tsujan** [reply]
> lxqt-panel uses 70M…

These numbers depend on so many factors.

For example, here, under LXQt+labwc, lxqt-panel starts with ~47 MiB and may reach 60 after two weeks, while under LXQt+kwin_wayland, it starts with ~37 MiB and may reach the same value. pcmanfm-qt starts with a low value like 30 MiB and may reach 200-350 after two weeks, because file management depends on various factors, and file caches grow according to global settings like `swappiness` and `vfs_cache_pressure`; etc. X11 and Wayland numbers are also different. For me, Wayland is lighter and more responsive.

**Reply by sonichy** [reply]
In my Arch
![lxqt_memory](https://github.com/user-attachments/assets/0a705314-d245-4900-bd39-28f884a03fc3)
